src/components/Repository.js

```diff
@@ -125,10 +163,15 @@ function Repository({match}) {
               <a rel="noreferrer" target="_blank" href={`https://codetriage.com/${nameWithOwner}`}>
                 <Button primary>Set up CodeTriage</Button>
               </a>
+              {isForked ?
+                <a rel="noreferrer" target="_blank" href={`https://github.com/${user.login}/${repoName}`}>
+                  <Button disabled={isForkLoading}>Go to fork</Button>
+                </a> :
+                <Button disabled={isForkLoading} onClick={forkRepository}>Fork {humanizeNumber(forks.totalCount)}</Button>}
```

Comment:
I think it would be better to display parenthesis around the number, it kinda doesn't make sense if there's the text Fork 204, I think Fork (204) would be more descriptive.
```suggestion
                <Button disabled={isForkLoading} onClick={forkRepository}>Fork ({humanizeNumber(forks.totalCount)})</Button>}
```

Reply:
Yeah, good point, if the maintainers agree they can commit the suggestion

Reply:
Sounds good for me! WDYT @bdougie?

Reply:
Instead of the number use the fork icon. The number doesn't add value or context to the button.